packages/medusa/src/api/routes/admin/product-categories/get-product-category.ts

```diff
@@ -0,0 +1,70 @@
+import { Request, Response } from "express"
+
+import ProductCategoryService from "../../../../services/product-category"
+import { defaultAdminProductCategoryRelations } from "."
+
+/**
+ * @oas [get] /product-categories/{id}
+ * operationId: "GetProductCategory"
```

Comment:
```suggestion
 * operationId: "GetProductCategoriesProductCategory"
```

Reply:
We have an entry in our [Code Conventions doc](https://www.notion.so/medusajs/Code-Conventions-dcdeccf9eca14ea1975b19b92a44c94d#18732732166741c79e943ce48600ca50). Let me know if it makes sense. Otherwise, we might have to revisit to make it more clear :)